assets/input/basic.css

```diff
@@ -87,7 +87,8 @@ textarea[data-valid="false"]:focus {
   @apply text-grey-dark;
 }
 
-input.input_left {
+input.input_left,
+div.input_left {
```

Comment:
Do we need a `div` tag here? We use the `input` tag to ensure specificity and catch all the browser overrides and such.

Reply:
I should have used input you're right ! I am changing it